src/test/scala/com/high-performance-spark-examples/CustomDataFrameSuiteBase.scala

```diff
@@ -0,0 +1,12 @@
+package com.highperformancespark.examples
+
+import com.holdenkarau.spark.testing.DataFrameSuiteBase
+
+trait CustomDataFrameSuiteBase extends DataFrameSuiteBase {
+
+  override def beforeAll() {
+    super.beforeAll()
+    sc.setLogLevel(org.apache.log4j.Level.WARN.toString)
```

Comment:
I think we can do this with a log4j properties file instead of changing the base class here.